api/python_api/kubeflow_trainer_api/models/io_k8s_api_core_v1_pod_certificate_projection.py

```
@@ -0,0 +1,97 @@
+    Do not edit the class manually.
+"""  # noqa: E501
+
+
+from __future__ import annotations
+import pprint
+import re  # noqa: F401
+import json
+
+from pydantic import BaseModel, ConfigDict, Field, StrictInt, StrictStr
+from typing import Any, ClassVar, Dict, List, Optional
+from typing import Optional, Set
```

Flake8 (7.3.0)

[error] redefinition of unused 'Optional' from line 21

(F811)